internal/sim/simulator.go

```diff
@@ -598,6 +598,9 @@ func (r *Results) Mermaid() string {
 		case EventDeliverError:
 			call := calls[x.SpanID]
 			fmt.Fprintf(&b, "    note right of %s%d: [%d:%d] RemoteCallError\n", call.Caller, call.Replica, x.TraceID, x.SpanID)
+		case EventPanic:
+			// TODO(mwhittaker): Show stack trace?
```

Comment:
I think we should.

Reply:
Done! The stack traces are a bit ugly for now, but we can improve them later.

![83JDoBNZLuXJLGp](https://github.com/ServiceWeaver/weaver/assets/3654277/137215fc-d288-4c9c-aca7-774a01e34ce6)